### PR TITLE
Update README.md grammar

### PR DESCRIPTION
**Pull request by Yeah-Jack**
## Type of change

- [x] Bug fix
- [ ] New feature

## Description

Improve the README grammar.

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have added comments to my code in more complex areas.
- [x] I have tested the code in both development and production environments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,28 +29,28 @@ Follow the [guide](https://meteorclient.com/faq/installation) on the wiki.
...
 - IDE or system-related files should be added to the `.gitignore`, never committed in pull requests.
 - In general, check existing code to make sure your code matches relatively close to the code already in the project.
 - Favour readability over compactness.
 - If you need help, check out the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html) for a reference.
 
 ## Bugs and Suggestions
-Bug reports and suggestions should be made in this repo's [issue tracker](https://github.com/MeteorDevelopment/meteor-client/issues) using the templates provided.  
-Please provide as much information as you can to best help us understand your issue and give a better chance of it being resolved.
+Bug reports and suggestions should be made in this repo's [issue tracker](https://github.com/MeteorDevelopment/meteor-client/issues) using the templates provided.  
+Please provide as much information as you can to best help us understand your issue and give us a better chance of it being resolved.
```
Comment (RacoonDog) [inline README.md:40]: "us" doesn't fit here

File: README.md [hunk 2]
```diff
@@ -29,28 +29,28 @@ Follow the [guide](https://meteorclient.com/faq/installation) on the wiki.
...
 
 ## Donations
-All of our work is completely free and non-profit (donations pay only for hosting costs), therefore we are very grateful for all donations made to support us in running our community.  
-Donations can be made via our [website](https://meteorclient.com/donate) and the minimum amount to get donor benefits is €5.  
-You will be rewarded with a role on our Discord server and a customisable in-game cape.  
+All of our work is completely free and non-profit (donations pay only for hosting costs); therefore, we are very grateful for all donations made to support us in running our community.  
+Donations can be made via our [website](https://meteorclient.com/donate) and the minimum amount to get donor benefits is €5.  
+You will be rewarded with a role on our Discord server and a customizable in-game cape.
```
Comment (RacoonDog) [inline README.md:45]: both "customisable" and "customizable" are valid (and i personally prefer the former)

File: README.md [hunk 3]
```diff
@@ -4,17 +4,17 @@
 </p>
 
 <h1 align="center">Meteor</h1>
-<p align="center">A Minecraft Fabric Utility Mod for anarchy servers.</p>
+<p align="center">A Minecraft Fabric Utility Mod for Anarchy Servers </p>
```
Comment (RacoonDog) [inline README.md:7]: it'd be better to give this proper capitalization instead of starting each word with a capital letter

**Review by RacoonDog — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Yeah-Jack) [reply README.md:40]: Why that?

**Pull request closed without merging**